src/linux/component.rs

```diff
@@ -11,8 +11,12 @@ use std::fs::{read_dir, File};
 use std::io::Read;
 use std::path::{Path, PathBuf};
 
+#[cfg(feature = "serde")]
+use serde::Serialize;
```

Comment:
You could remove this import if you used `derive(serde::Serialize)` directly.

Reply:
I can make that change tonight. Would you prefer to directly reference it and remove the configuration flag for usings at the top in all files?